### ISSUE TITLE
about the screen

### ISSUE THREAD

**Issue by DrBomer** [dcpedit/dissatisfaction30#6]
hello, I want to know that whether the screen image can be changed. If yes please tell me how. Thank you

**Comment by dcpedit** [comment]
Yes, you can follow the instructions for my Red Herring keyboard (Change OLED Graphic): https://github.com/dcpedit/redherring/tree/main/firmware#change-oled-graphic

You'll need to change the following line since the screen is a different resolution: (32x128)/8
```
#define LOGO_SIZE 512
```
